lib/ansible/modules/net_tools/nios/nios_network.py

```diff
@@ -87,9 +87,9 @@
 '''
 
 EXAMPLES = '''
-- name: configure a network
+- name: configure a networkv4/v6
```

Comment:
If this is an example for both v4 and v6, I don't think the description change is necessary. If separate examples are more realistic, then two examples with ` (v4)` and ` (v6)` suffix seems more appropriate

Reply:
It'll surely make things more clearer with separate example for v4 and v6, so I can add it likewise.

Reply:
Yeah, you don't have to duplicate everything, just show at least one example for each to indicate that it works for both ipv4 & ipv6

Reply:
Done